File: app/src/main/java/fr/paug/androidmakers/ui/adapter/RoomSchedule.java
package fr.paug.androidmakers.ui.adapter;

import android.support.annotation.NonNull;

import java.util.List;

public class RoomSchedule implements Comparable<RoomSchedule> {

    private final int mRoomId;
    private final String mTitle;
    @NonNull
    private final List<ScheduleSession> mScheduleSessions;

    public RoomSchedule(int roomId, String title, @NonNull List<ScheduleSession> scheduleSessions) {
        mRoomId = roomId;
        mTitle = title;
        mScheduleSessions = scheduleSessions;
    }

    public int getRoomId() {
        return mRoomId;
    }

    public String getTitle() {
        return mTitle;
    }

    @NonNull
    public List<ScheduleSession> getItems() {
        return mScheduleSessions;
    }

    @Override
    public int compareTo(@NonNull RoomSchedule o) {
        return mRoomId - o.mRoomId;
    }

    @Override
    public String toString() {
        return "RoomSchedule{" +
                "mRoomId=" + mRoomId +
                ", mTitle='" + mTitle + '\'' +
                ", mItems=" + mScheduleSessions +
                '}';
    }

}

File: app/src/main/java/fr/paug/androidmakers/ui/adapter/RoomSchedule.kt
package fr.paug.androidmakers.ui.adapter

import fr.paug.androidmakers.util.EmojiUtils.getLanguageInEmoji
import fr.paug.androidmakers.ui.adapter.RoomSchedule
import fr.paug.androidmakers.ui.adapter.ScheduleSession
import fr.paug.androidmakers.model.ScheduleSlot
import fr.paug.androidmakers.util.EmojiUtils
import androidx.fragment.app.FragmentPagerAdapter
import fr.paug.androidmakers.R
import fr.paug.androidmakers.ui.fragment.VenueConferenceFragment
import fr.paug.androidmakers.ui.fragment.VenueAfterPartyFragment
import fr.paug.androidmakers.ui.fragment.VenueFloorPlanFragment
import com.google.android.material.floatingactionbutton.FloatingActionButton
import android.widget.Checkable
import fr.paug.androidmakers.ui.util.CheckableFloatingActionButton

class RoomSchedule(val roomId: Int, val title: String, val items: List<ScheduleSession>) :
    Comparable<RoomSchedule> {
    override fun compareTo(o: RoomSchedule): Int {
        return roomId - o.roomId
    }

    override fun toString(): String {
        return "RoomSchedule{" +
                "mRoomId=" + roomId +
                ", mTitle='" + title + '\'' +
                ", mItems=" + items +
                '}'
    }
}